packages/babel-parser/test/fixtures/es2017/async-functions/invalid-escape-export-dflt-async-function/output.json

```diff
@@ -0,0 +1,64 @@
+{
+  "type": "File",
+  "start":0,"end":50,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":50}},
+  "errors": [
+    "SyntaxError: Missing semicolon (1:25)",
+    "SyntaxError: 'await' is only allowed within async functions (1:41)",
+    "SyntaxError: Missing semicolon (1:46)"
```

Comment:
The second error seems to contradict with the third error:
```
export default \u0061sync function y() { await x }
```
If we parse `await x` as an await expression and throws recoverable errors, we should not complain missing semicolon after `await`.

Reply:
Even if we add the semicolon, in a module we still get the same error:

```js
function y() {
  await
}
```

This is because `await` can only be used inside async functions if `sourceType: "module"`, but then we recover by parsing it as an identifier as if `sourceType` was `"script"`.